warehouse/query-core/src/main/java/datawave/query/jexl/JexlASTHelper.java

```diff
@@ -207,30 +204,28 @@ private static ASTJexlScript parseQueryWithBackslashes(String query, Parser pars
         while (!workingStack.isEmpty()) {
             JexlNode node = workingStack.pop();
             
-            if (node.image != null) {
-                int numToReplace = StringUtils.countMatches(node.image, placeholder);
+            if (node instanceof ASTStringLiteral) {
```

Comment:
We used to parse the image of all nodes.  Are we we sure ASTStringLiteral is the only one we need to handle?

Reply:
The image parameter no longer exists.  

For context, there are 3 kinds of nodes that contain a literal now: ASTRegexLiteral (new, and unused by us), ASTStringLiteral, ASTNumberLiteral. 

This check was added by me a while ago to fix the right hand (String literal) side of EQ, ER, or NR nodes whose image contain backslashes so that we can escape them properly.  At the time, keying off of the `image` parameter was more convenient than checking to see whether the node was an ASTStringLiteral.  I can say pretty confidently that I am not interested in modifying anything that is not an ASTStringLiteral with this segment of code.  If you can think of something else that needs to be included here, let me know, but nothing comes to my mind.

Reply:
I see now.  np